addons/controllers/clusterbootstrap_controller.go

```diff
@@ -509,6 +517,13 @@ func (r *ClusterBootstrapReconciler) createOrPatchPackageInstallOnRemote(cluster
 				Prereleases: &versions.VersionSelectionSemverPrereleases{},
 			},
 		}
+
+		if remotePkgi.Labels == nil {
+			remotePkgi.Labels = map[string]string{}
+		}
+		remotePkgi.Labels[types.ClusterNameLabel] = cluster.Name
+		remotePkgi.Labels[types.ClusterNamespaceLabel] = cluster.Namespace
```

Comment:
question: There seems various way to add labels, I believe they are essentially the same(maybe I am wrong). Make sure we add some tests to verify the labels.

```
remotePkgi.SetLabels()
remotePkgi.ObjectMeta.Labels
remotePkgi.ObjectMeta.SetLabels()
```

Reply:
Changed to one liners L392 & L502. Added test to make sure labels are set in ```clusterbootstrap_test.go```